### PR TITLE
[PHP] stdClass Serialization

### PR DESCRIPTION
**Pull request by collinkrawll**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

This PR implements #7327 ([PHP] - Do not treat \stdClass as a Model)

/cc @jebentier (2017/07) @dkarlovi (2017/07) @mandrean (2017/08) @jfastnacht (2017/09) @ackintosh (2017/09)

### CODE REVIEW

File: samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php [hunk 1]
```diff
@@ -1075,6 +1075,237 @@ protected function fakeOuterStringSerializeRequest($body = null)
...
+                    $response->getHeaders(),
+                    $response->getBody()
+                );
+            }
+
+            return [null, $statusCode, $response->getHeaders()];
+
+        } catch (ApiException $e) {
+            switch ($e->getCode()) {
+            }
```
Comment (olleolleolle) [inline samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php]: Does this `switch` do anything?

File: samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php [hunk 2]
```diff
@@ -1075,6 +1075,237 @@ protected function fakeOuterStringSerializeRequest($body = null)
...
+            throw $e;
+        }
+    }
+
+    /**
+     * Operation testBodyWithQueryParamsAsync
+     *
+     * 
+     *
```
Comment (olleolleolle) [inline samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php]: There's extra whitespace lines here. Can they be removed?

File: samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php (hunk 1 above)
Comment (collinkrawll) [reply samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php]: Doesn't look like it does anything but that file is actually auto generated by the bin scripts. The only real change in this PR is in [this commit](https://github.com/swagger-api/swagger-codegen/pull/8144/commits/215e6eaf5606c828096f9626ea19b6a21d2e0062). The other two commits are unit testing and a re-run of the sample generator (which produced this code).

File: samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php (hunk 2 above)
Comment (collinkrawll) [reply samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php]: Yeah, I'd imagine so but that file is actually auto generated by the bin scripts. The only real change in this PR is in [this commit](https://github.com/swagger-api/swagger-codegen/pull/8144/commits/215e6eaf5606c828096f9626ea19b6a21d2e0062). The other two commits are unit testing and a re-run of the sample generator (which produced this code).

**Comment by HugoMario** [comment]
please build

**Comment by HugoMario** [comment]
thanks a lot @collinkrawll sorry for this long delay

**Pull request merged**